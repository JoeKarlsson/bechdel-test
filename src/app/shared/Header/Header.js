import React from 'react';
import {
	NavLink,
} from 'react-router-dom';
import './Header.scss';

const activeStyles = {
	color: 'red',
};

const Header = () => {
	return (
		<header className="header_bar">
			<NavLink to="/" className="header_logo">
				bechdel.io
			</NavLink>
			<ul className="header_nav">
				<li>
					<NavLink to="/about" activeStyle={activeStyles}>about</NavLink>
				</li>
				<li>
					<NavLink to="/case-study"activeStyle={activeStyles}>case study</NavLink>
				</li>
			</ul>
		</header>
	);
};

export default Header;
