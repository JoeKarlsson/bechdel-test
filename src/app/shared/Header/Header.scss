$purple: purple;
$white: white;

.header_bar {
	width: 100%;
	background-color: $purple;
	padding-top: 2%;
	padding-bottom: 2%;
	font-size: 1.25rem;
	display: flex;
	flex-direction: row;

	a {
		color: $white;
	}
}

ul {
	margin: 0;
	padding: 0;
	list-style: none;
	display: flex;
	flex-direction: row;

	li {
		padding: 0.5rem 1rem;
	}
}

.header_logo {
	padding: 0.5rem 1rem;
	padding-left: 5%;
	flex: 1 0 auto;
	order: 10;
	font-weight: 200;
	font-size: large;
}

.header_nav {
	padding-right: 5%;
	flex: 0 1 auto;
	order: 20;
	list-style: none;
}
