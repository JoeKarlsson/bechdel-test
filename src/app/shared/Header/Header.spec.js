import React from 'react';
import Header from './Header';

describe('Header Page', () => {
	describe('rendering', () => {
		it('match the snapshot', () => {
			const wrapper = shallow(<Header />);
			expect(wrapper).toMatchSnapshot();
		});
	});
});
