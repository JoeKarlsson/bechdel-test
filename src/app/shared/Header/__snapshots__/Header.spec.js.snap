// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Page rendering match the snapshot 1`] = `
ShallowWrapper {
  "length": 1,
}
`;
